### DISCUSSION TITLE
Time-updated derivative field

### DISCUSSION THREAD

**Discussion by webwurm** [AntlerVC/firetable#954] [Support / Q&A] [answered]
I have a short question if this is possible in Rowy:
I have a Date/time-field "ValidUntil" and a derivative field "Expired" (bool).
"Expired" compares the ValidUntil-field with the current Date/Time and returns either False or True.

Is it somehow possible that the Expired-field always has the correct status, whenever I query it?
Any ideas on how to implement that?

Thanks a lot!

**Comment by shamsmosowi** [comment, marked as answer]
not sure if what you're describing is possible, can you query the "ValidUntil" field with `=<` current Date instead
